### ISSUE TITLE
Rename the project and website

### ISSUE THREAD

**Issue by wschella** [ZeusWPI/aichallenge#110]
Due to copyright claims, we can no longer use Badass Battle Bots.
When a new name has been decided, we should replace the original everywhere.

**Comment by iasoon** [comment]
Can this be closed?

**Comment by wschella** [comment]
No, as of now, internally 'battlebots' is still used, for consistency this should be renamed as well.
